src/devtools/devtools.tsx

```diff
@@ -540,12 +540,24 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
               alignItems: 'center',
             }}
           >
-            <Logo
-              aria-hidden
+            <button
```

Comment:
can you please add a test that makes sure clicking the button actually closes the devtools?

Reply:
Added test case for it. 👍